store.go

```diff
@@ -2930,15 +2930,40 @@ func (s *store) Unmount(id string, force bool) (bool, error) {
 }
 
 func (s *store) Changes(from, to string) ([]archive.Change, error) {
```

Comment:
I couldn’t find  anything, apart from `cmd` here, that is _using_ `Changes`. It’s tempting to think that it should be deprecated or somehow made private instead of expanding functionality. That’s probably not this PR.